tools/integration-test/src/tests/ordered_channel.rs

```diff
@@ -2,10 +2,11 @@ use ibc_test_framework::ibc::denom::derive_ibc_denom;
 use ibc_test_framework::prelude::*;
 use ibc_test_framework::util::random::random_u64_range;
 
-#[test]
-fn test_ordered_channel() -> Result<(), Error> {
-    run_binary_channel_test(&OrderedChannelTest)
-}
+// #[test]
```

Comment:
Question: Is this test meant to be commented out?

Reply:
Nope, good catch!